Week-10/shortify/src/Home.js

```diff
@@ -0,0 +1,75 @@
+import {useState} from 'react';
+import {GoArrowRight} from 'react-icons/go';
+import homeImg from './assets/homePageImg.svg';
+import ShortList from './components/ShortList';
+import LinearLoader from './components/LinearLoader';
+import axios from 'axios';
+import './styles/Home.css'
+
+
+function Home() {
+  const [state, setState] = useState(
+    {
+      list:[],
+      loading:false,
+    }
+  )
+
+  function handleSubmit(e){
+    e.preventDefault();
```

Comment:
create more functions for making api call, api calls should be done inside services folder